src/Lumina/GameData.cs

```diff
@@ -190,10 +190,11 @@ public GameData( string dataPath, ILogger logger, LuminaOptions? options = null!
         /// Load a defined file given a filesystem path
         /// </summary>
         /// <param name="path">A relative or absolute path to the file to load</param>
+        /// <param name="origPath">The original file path in SqPack, required for reading some files (e.g. materials)</param>
         /// <typeparam name="T">The type of <see cref="FileResource"/> to load the raw file in to</typeparam>
         /// <returns>The requested file if found, null if not</returns>
         /// <exception cref="FileNotFoundException">The path given doesn't point to an existing file</exception>
-        public T GetFileFromDisk< T >( string path ) where T : FileResource
+        public T GetFileFromDisk< T >( string path, string? origPath = null ) where T : FileResource
```

Comment:
I don't know if that's a concern, but this breaks backwards compatibility by the way.

IIRC, creating `Func<string, T>` delegates out of this method won't work any more, also the code that depends on it will need a recompile.